src/App.js

```diff
@@ -52,6 +53,13 @@ function Rutas() {
         <AltaModificacionEspacio titulo={'Modificar espacio'} />
       </PrivateRoute>
 
+      <PrivateRoute
+        path="/misActividades"
+        rolesPermitidos={['asistente', 'admin']}
```

Comment:
No tiene sentido esta pantalla para el admin.

```suggestion
        rolesPermitidos={['asistente']}
```